.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,23 @@
+name: C/C++ CI
```

Comment:
Thanks for your PR, very happy to see some progress in this area. Since I am not the most familiar with the C extensions, would you mind adding a comment to specify what this improves on?